content/java/concepts/math-methods/terms/to-int-exact/to-int-exact.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.toIntExact()'
+Description: 'Returns the integer value of a long type argument. An exception is thrown if the value of argument overflows that of an int.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.toIntExact()`** method returns the integer value of a `long` type argument. An [exception](https://www.codecademy.com/resources/docs/java/errors) is thrown if the value of argument overflows that of an `int`.
+
+## Syntax
+
+```pseudo
+Math.toIntExact(long val)
+```
+
+The return type of the `.toIntExact()` method is an `int` type value. The data type of the `val` parameter is `long`.
+
+If the value of `val` is greater than 2147483647L or is lesser than -2147483648L then an exception of following manner will be thrown:
```

Comment:
"...or is ~lesser~ less than -2147483648L, then ~an exception of the following manner~ the following exception will be thrown."